tardis/montecarlo/montecarlo_state.py

```diff
@@ -0,0 +1,334 @@
+import warnings
```

Comment:
This might be a bit only a minor detail, but I would name this file ``montecarlo_transport_state.py`` as it only contains the ``MonteCarloTransportState`` class.

Reply:
Agreed